### PR TITLE
adding some php doc blocks

### PR DESCRIPTION
**Pull request by spekulatius**

### CODE REVIEW

File: SensioLabs/Security/SecurityChecker.php [hunk 1]
```diff
@@ -17,9 +17,19 @@
...
+     */
     private $vulnerabilityCount;
+    
+    /**
+     * @var CrawlerInterface
+     */
     private $crawler;
 
+    /**
+     * @param CrawlerInterface $crawler
```
Comment (GrahamCampbell) [inline SensioLabs/Security/SecurityChecker.php]: wrong

Comment (GrahamCampbell) [inline SensioLabs/Security/SecurityChecker.php]: missing |null

Comment (GrahamCampbell) [inline SensioLabs/Security/SecurityChecker.php]: also, this phpdoc isn't meaningful, which was why it was omitted in the first place i assume

**Comment by spekulatius** [comment]
okay, I'll close this.

**Pull request closed without merging**